domains/laman.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Derogater",
+        "email": "thelazybloggers@gmail.com"
+    },
+    "record": {
+        "TXT": "txt"
```

Comment:
txt alone does nothing

Reply:
hey, how should i set this up then, can you give an example or you can add as according to the standards on your own accord just keep the name same(in case you are to do it), else please tell me what to do, please.